.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: Please describe the behavior you expected
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual Behavior
+      description: Please describe the behavior that actually happened
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps To Reproduce
+      description: Please describe the steps required to reproduce this issue
+    validations:
+      required: true
+  - type: dropdown
+    id: featureform-mode
+    attributes:
+      label: What mode are you running Featureform in?
+      options:
+        - Local
+        - Hosted
+    validations:
+      required: true
+  - type: dropdown
+    id: python-verison
+    attributes:
+      label: What version of Python are you running?
```

Comment:
Should we add a description that states we only support these versions?

Reply:
Do we need it? The drop down forces you to choose one of the version and you can only install it on those versions anyways.